src/metis/cli/utils.py

```diff
@@ -106,6 +106,20 @@ def pretty_print_reviews(results, quiet=False):
                         f"    [cyan]Line number:[/cyan] {r['line_number']}",
                         quiet,
                     )
+                if r.get("cwe"):
+                    print_console(f"    [red]CWE:[/red] {r['cwe']}", quiet)
+                severity = r.get("severity")
+                if severity:
+                    severity_color = {
+                        "Low": "green",
+                        "Medium": "yellow",
+                        "High": "red",
+                        "Critical": "magenta",
+                    }.get(severity, "bright_black")
+                    print_console(
+                        f"    [bright_black]Severity:[/bright_black] [bold {severity_color}]{severity}[/bold {severity_color}]",
+                        quiet,
+                    )
                 if r.get("reasoning"):
                     print_console(
                         f"    [white]Why:[/white] {escape(r['reasoning'])}", quiet
```

Comment:
More walrus operator.

```suggestion
                if reasoning := r.get("reasoning"):
                    print_console(
                        f"    [white]Why:[/white] {escape(reasoning)}", quiet
```